File: src/HW1/Calculator.cs
namespace HW1
{
    public class Calculator
    {
        public static int Calculate(int val1, string operation, int val2)
        {
            var result = operation switch
                         {
                             "+" => val1 + val2,
                             "-" => val1 - val2,
                             "*" => val1 * val2,
                             "/" => val1 / val2,
                             _   => 0
                         };
            return result;
        }
    }
}

File: src/HW3/Calculator.fs
namespace HW3

module Calculator =
    let Calculate (val1: int) (operation: string) (val2: int):int =
        match operation with
        | "+" -> val1 + val2
        | "-" -> val1 - val2
        | "*" -> val1 * val2
        | "/" -> val1 / val2
        | _ -> 0